frontend/public/components/search-filter-dropdown.tsx

```diff
@@ -14,17 +17,19 @@ export const SearchFilterDropdown: React.SFC<SearchFilterDropdownProps> = (props
 
   const { onChange, nameFilterInput, labelFilterInput } = props;
 
+  const { t } = useTranslation();
+
   const onToggle = (open: boolean) => setOpen(open);
   const onSelect = (event: React.SyntheticEvent) => {
     setSelected((event.target as HTMLInputElement).name as searchFilterValues);
     setOpen(!isOpen);
   };
   const dropdownItems = [
     <DropdownItem key="label-action" name={searchFilterValues.Label} component="button">
-      {searchFilterValues.Label}
+      {t(searchFilterValues.Label)}
```

Comment:
Can you confirm that this doesn't break selection of these items? How do we key on these values internally if we're translating the dropdown values?

Reply:
It keys off of the name value, which I didn't touch.

```
const onSelect = (event: React.SyntheticEvent) => {
    setSelected((event.target as HTMLInputElement).name as searchFilterValues);
    setOpen(!isOpen);
  };
```

Reply:
Works fine on my end.